.github/dependabot.yml

```diff
@@ -7,8 +7,7 @@ updates:
     directory: "/tdrs-frontend"
     schedule:
       interval: "daily"
-    target-branch: "dependabot-mass-merge"
-    open-pull-requests-limit: 50
+    target-branch: "raft-tdp-main"
```

Comment:
Sets dependabot config back to normal so we can process future updates individually.